### PR TITLE
config: tgl-cavs: add smart_amp module config

### PR DESCRIPTION
**Pull request by aiChaoSONG**
This patch adds module config for smart_amp.

Signed-off-by: Chao Song <chao.song@linux.intel.com>

### CODE REVIEW

File: config/tgl-cavs.toml [hunk 1]
```diff
@@ -266,3 +266,22 @@ count = 9
 		20, 0, 0, 0, 12832, 20881000, 0, 0, 0, 20881, 0,
 		21, 0, 0, 0, 12832, 23431000, 0, 0, 0, 23431, 0,
 		22, 0, 0, 0, 12832, 30471000, 0, 0, 0, 30471, 0]
+
+	[[module.entry]]
+	name = "SMTAMP"
+	uuid = "167A961E-8AE4-11EA-89F1-000C29CE1635"
```
Comment (plbossart) [inline config/tgl-cavs.toml:272]: it should be clearer that this is a test module....

**Review by plbossart — DISMISSED** [review]

File: config/tgl-cavs.toml (hunk 1 above)
Comment (lgirdwood) [reply config/tgl-cavs.toml:272]: @aiChaoSONG change the name and add some toml comments to indicate this is a testing module

Comment (aiChaoSONG) [reply config/tgl-cavs.toml:273]: updated! the max len for the module name is 8(include ending '\0'), so I will call it SMATEST

**Comment by aiChaoSONG** [comment]
@lgirdwood ping?

**Review by lgirdwood — APPROVED** [review]

**Comment by lgirdwood** [comment]
@plbossart good now ?

**Review by lgirdwood — APPROVED** [review]

**Pull request merged**